structure_tests/drivers/docker_driver.go

```diff
@@ -311,3 +305,32 @@ func (d *DockerDriver) exec(t *testing.T, env []string, command []string) (strin
 
 	return stdout.String(), stderr.String(), exitCode
 }
+
+func (d *DockerDriver) GetConfig(t *testing.T) (unversioned.Config, error) {
```

Comment:
This function looks basically the same as the tardriver version, can you refactor to share the implementation at all?

Reply:
since the structs returned by both implementations are inherently different it's kind of hard to consolidate this code. eventually I can switch container-diff to populate the docker client `Config` object and then these can be combined.